src/annotator/anchoring/html.js

```diff
@@ -64,22 +67,25 @@ export function anchor(root, selectors, options = {}) {
   let promise = Promise.reject('unable to anchor');
 
   if (range) {
+    const range_ = range;
```

Comment:
This `const` alias is needed as otherwise TS complains the variable might have changed inside the `catch` callback, since it is declared with `let` and could potentially be re-assigned before the callback runs.

This whole function could be reworked to avoid the unusual control flow here, but that's out of scope for this PR.